inlong-sort/sort-formats/format-inlongmsg-base/pom.xml

```diff
@@ -87,6 +87,18 @@
             <scope>provided</scope>
         </dependency>
 
+        <dependency>
+            <groupId>org.apache.flink</groupId>
+            <artifactId>flink-table-runtime-blink_${flink.scala.binary.version}</artifactId>
```

Comment:
please add all the new dependencies in notice and dependency file in /licenses/inlong-sort

Reply:
please refer to https://github.com/apache/incubator-inlong/pull/4158/files#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8